evalml/utils/nullable_type_utils.py

```diff
@@ -57,16 +58,31 @@ def _downcast_nullable_y(y, handle_boolean_nullable=True, handle_integer_nullabl
         y with any incompatible nullable types downcasted to compatible equivalents.
     """
     if y.ww.schema is None:
-        y = ww.init_series(y)
+        if isinstance(y, pd.DataFrame):
+            y.ww.init()
+        else:
+            y = ww.init_series(y)
```

Comment:
It's possible to initialize woodwork on a Series with `ww.init()` as well, we can simplify this to just call that regardless instead of splitting it up!